plugins/gtav/gtav.cpp

```diff
@@ -3,118 +3,213 @@
 // that can be found in the LICENSE file at the root of the
 // Mumble source tree or at <https://www.mumble.info/LICENSE>.
 
-#define MUMBLE_ALLOW_DEPRECATED_LEGACY_PLUGIN_API
-#include "mumble_legacy_plugin.h"
+#include "MumblePlugin_v_1_0_x.h"
 
-#include "mumble_positional_audio_main.h"  // Include standard positional audio header.
-#include "mumble_positional_audio_utils.h" // Include positional audio header for special functions, like "escape".
+#include "ProcessWindows.h"
+#include "mumble_positional_audio_utils.h"
 
-#include <algorithm> // Include algorithm header for the game version detector
+#include <cstring>
 
-static int fetch(float *avatar_pos, float *avatar_front, float *avatar_top, float *camera_pos, float *camera_front,
-				 float *camera_top, std::string &context, std::wstring &identity) {
+static std::unique_ptr< ProcessWindows > proc;
+static procptr_t memory_offset = 0;
```

Comment:
What about:

```suggestion
static procptr_t moduleBase = 0;
```

?